all/HOPE-6--Per-Interface-Defaults.md

```diff
@@ -36,7 +38,7 @@ The defaults would also be able to be applied to parameters whose endpoints have
 
 ```python
 @hug.http()
-@hug.cli(defaults={'user': getpass.getuser()})
+@hug.cli(user=getpass.getuser())
```

Comment:
Another option would be to use a decorator namespace, e.g. `@hug.cli.default(user=getpass.getuser())`.

Not yet sure which option is better. `hug.cli.default` might be more testable, and might be easier to document (i.e. use any kwargs you want). That said, it's implying addition of an interface. Might need to accommodate use of both `@hug.cli` and `@hug.cli.default`